media/css/protocol/components/split.scss

```diff
@@ -7,3 +7,12 @@ $image-path: '/media/protocol/img';
 
 @import '~@mozilla-protocol/core/protocol/css/includes/lib';
 @import '~@mozilla-protocol/core/protocol/css/components/split';
+
+// Issue 11457
+
+@media screen and (max-width: $screen-md) {
```

Comment:
initially, I had this selector as `.mzp-t-split-nospace.t-mobile-nospace` but ran into an issue with http://localhost:8000/en-US/firefox/pocket/, where the larger screen had a split section scrollbar because I was overriding some custom bottom padding

it works fine with a single class, but I added the media query too because it avoids redundant style rules and prevents any unintentional overrides on larger screens